pyEpiabm/pyEpiabm/property/personal_foi.py

```diff
@@ -0,0 +1,54 @@
+#
+# Calculate infectiousness and susceptibility for an individual
+#
+
+
+class PersonalInfection:
+    """Class to calculate the infectiousness and susceptibility
+    parameters an individual.
```

Comment:
```suggestion
    parameters of a :class:`Person`.
```
This might work as well, and should create a hyperlink in the docs